examples/service.json

```diff
@@ -3,6 +3,7 @@
   "kind": "Service",
   "apiVersion": "v1beta1",
   "port": 8000,
+  "containerPort": "http",
```

Comment:
k8s has a nifty IntOrString type. that's what the type of containerPort is:

https://github.com/GoogleCloudPlatform/kubernetes/blob/release-0.14/pkg/api/v1beta1/types.go#L739


On Tue, May 5, 2015 at 9:12 PM, Connor Doyle <notifications@github.com>
wrote:

> In examples/service.json
> <https://github.com/mesosphere/kubernetes-mesos/pull/254#discussion_r29728743>
> :
>
> > @@ -3,6 +3,7 @@
> >    "kind": "Service",
> >    "apiVersion": "v1beta1",
> >    "port": 8000,
> > +  "containerPort": "http",
>
> Is this value supposed to be an integer?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mesosphere/kubernetes-mesos/pull/254/files#r29728743>.
>